web/client/selectors/mapInfo.js

```diff
@@ -100,7 +100,8 @@ export const identifyOptionsSelector = createStructuredSelector({
     format: generalInfoFormatSelector,
     map: mapSelector,
     point: clickPointSelector,
-    currentLocale: currentLocaleSelector
+    currentLocale: currentLocaleSelector,
+    maxItems: (state) => get(state, "mapInfo.configuration.maxItems", 10)
```

Comment:
better to not add a default value if since now it was missing

```suggestion
    maxItems: (state) => get(state, "mapInfo.configuration.maxItems")
```